src/config.cc

```diff
@@ -1113,13 +1109,13 @@ Class load()
   c.lastMainGroupId  = root.namedItem( "lastMainGroupId" ).toElement().text().toUInt();
   c.lastPopupGroupId = root.namedItem( "lastPopupGroupId" ).toElement().text().toUInt();
 
-  QDomNode popupWindowState = root.namedItem( "popupWindowState" );
+  QDomNode popupWindowState = root.namedItem( "popupWindowState2" );
```

Comment:
name changed.

Reply:
Yes, this is correct, because the popup as a whole is changed.